### PR TITLE
EMP Blast

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Edited the EMP animation to have a nice look.

## Why / Balance
It looked bad

## How to test
Spawn emp tools and have fun

## Media
Too large
https://discord.com/channels/1123826877245694004/1123826879892299783/1296215896402165821

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
:cl:
- tweak: EMP Pulse got a new visual look.

### CODE REVIEW

File: Content.Shared/_NF/Emp/Components/EmpBlastComponent.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    /// </summary>
+    [AutoNetworkedField]
+    public TimeSpan StartTime;
+
+    /// <summary>
+    ///     How long will animation play in seconds.
+    ///     Can be overridden by <see cref="Robust.Shared.Spawners.TimedDespawnComponent"/>.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float VisualDuration = 2f;
```
Comment (Cheackraze) [inline Content.Shared/_NF/Emp/Components/EmpBlastComponent.cs:23]: this feels really long to me. for a 'blast', the effect lasts a bit too long and makes the screen hard to see if there are more than 1 happening at once.

Comment (dvir001) [reply Content.Shared/_NF/Emp/Components/EmpBlastComponent.cs:23]: Can try it as 1 sec, the animation speed is fully under control.

**Comment by GreaseMonk** [comment]
do we want to show this blast when graphics is set to low ? just wondering

**Comment by dvir001** [comment]
> do we want to show this blast when graphics is set to low ? just wondering

Its fine

**Comment by dvir001** [comment]
I fixed the code to allow it to be shown with infinite range only when the EMP is 25+ (M_EMP) so you can see EMP coming from outside your PVS range.

**Review by GreaseMonk — APPROVED** [review]
It works

**Pull request merged**